public/assets/js/swiper.js

```diff
@@ -64,15 +58,15 @@ document.addEventListener('DOMContentLoaded', () => {
         },
     });
 
-    const insightMainSlider = new Swiper('.insights-slider', {
+    const insightMainSlider = new Swiper(".insights-slider", {
         loop: true,
         slidesPerView: 1,
         spaceBetween: 30,
         autoplay: {
             delay: 3000,
         },
         pagination: {
-            el: '.swiper-pagination',
```

Comment:
Not a big deal, but don't know why to change these quotes? It's recommended to follow same coding conventions within the project and we are using single quotes `''` for strings in PHP and JS.